test/jtreg-ext/requires/VMProps.java

```diff
@@ -527,16 +528,16 @@ protected String isCompiler2Enabled() {
     }
 
     /**
-     * A simple check for docker support
+     * A simple check for container support
      *
-     * @return true if docker is supported in a given environment
+     * @return true if container subsystem is supported in a given environment
      */
-    protected String dockerSupport() {
-        log("Entering dockerSupport()");
+    protected String containerSupport() {
+        log("Entering containerSupport()");
 
-        boolean isSupported = false;
+        boolean isSupported = true;
```

Comment:
This needs to be initialized to false.

Reply:
Good catch, thank you. This is a bug, a leftover from my experimentation.